primitives/Cargo.toml

```diff
@@ -42,6 +42,8 @@ smallstr = { version = "0.3.0", default-features = false, features = ["serde", "
 thiserror = { workspace = true, optional = true }
 displaydoc = { workspace = true }
 parking_lot = { workspace = true, optional = true }
+serde_json = { workspace = true }
```

Comment:
does this mean that this dependency can be removed from some crates, like `iroha_data_model`?

Reply:
Not yet. data-model still has a lot of dependencies on serde_json